app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt

```diff
@@ -25,14 +28,17 @@ class LoginPresenter @Inject constructor(private val userSession: UserSession, p
             errorHappened = true
         }
         if (!errorHappened) {
+            view?.toggleSpinnerVisibility(SHOW_SPINNER)
```

Comment:
since toggleSpinnerVisibility receive a boolean as a parameter, you can just just send !errorHappened as that boolean value

Reply:
to avoid calling twice view?.toggleSpinnerVisibility() and to avoid declaring boolean values as constants